src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py

```diff
@@ -252,6 +276,26 @@ def acr_build_task_update(cmd,  # pylint: disable=too-many-locals
     return build_task
 
 
+def acr_build_task_update_build(cmd,
+                                client,
+                                build_id,
+                                registry_name,
+                                no_archive=None,
+                                resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd.cli_ctx, registry_name, resource_group_name, BUILD_TASKS_NOT_SUPPORTED)
+
+    if no_archive is not None:
+        is_archive_enabled = not no_archive
+    else:
+        is_archive_enabled = None
```

Comment:
NIT: can use one line 
`is_archive_enabled = not no_archive if no_archive is not None else None`